packages/components/src/StoryFeed/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface StoryNewsroomData {
+  contractAddress: string;
+  multisigAddress: string;
+  charter: {
+    name: string;
+    newsroomUrl: string;
+    mission: {
+      purpose: string;
+    };
+    socialUrls?: {
+      twitter?: string;
+      facebook?: string;
+      email?: string;
+    };
+  };
+}
+
+export interface OpenGraphData {
+  article: {
+    published_time: string;
+  };
+  description: string;
+  images: {
```

Comment:
left this comment elsewhere, but makes more sense here. if this were to match how it's stored on the backend, images is actually an array of objects. so accessing the image would be images[0].url